value/src/main/java/com/google/auto/value/processor/AutoValueProcessor.java

```diff
@@ -216,6 +216,7 @@ private String generatedSubclassName(TypeElement type, int depth) {
 
     private ImmutableList<String> buildAnnotations(TypeSimplifier typeSimplifier) {
       ImmutableList.Builder<String> builder = ImmutableList.builder();
+      Set<Name> methodAnnotations = null;
```

Comment:
I think it is fair to assume that any public JDK class or interface that implements `equals(Object)` also implements a corresponding `hashCode()`. Otherwise it would be violating the contract of `hashCode()`.